src/Views/Format.elm

```diff
@@ -63,6 +64,48 @@ formatFoodSelectedImpactPerKg { trigram, unit, decimals } totalMass =
         >> formatRichFloat decimals (unit ++ "/kg")
 
 
+formatFoodSelectedImpactScore : Impact.Definition -> Mass -> Impacts -> Html msg
+formatFoodSelectedImpactScore { trigram } totalMass impacts =
+    let
+        ln =
+            logBase e
+
+        score =
+            impacts
+                |> Impact.perKg totalMass
+                |> Impact.getImpact trigram
+                |> Unit.impactToFloat
+                |> (\value ->
+                        -- See the documentation at https://fabrique-numerique.gitbook.io/ecobalyse/alimentaire/impacts-consideres/score-100
+                        (ln 2077 - ln value) / ln 2 * 20
+                   )
+                |> round
+                |> clamp 0 100
+
+        letter =
+            if score >= 80 then
+                "A"
+
+            else if score >= 60 then
+                "B"
+
+            else if score >= 40 then
+                "C"
+
+            else if score >= 20 then
+                "D"
+
+            else
+                "E"
```

Comment:
I couldn't find a requirement for rendering a letter, is that an informal one?

Reply:
yep, a discussion with Lisa about adding this as an experiment and base for future discussions